emr-etl-runner/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM jruby:9.1
+MAINTAINER Josh Cox <josh 'at' webhosting.coop>
+
+ENV PLAIN_OF_JARS=20170705 \
+LANG=en_US.UTF-8
+
+RUN apt-get update \
+&& apt-get install -yqq gettext-base mlocate \
```

Comment:
I think we can remove `mlocate` since the storage loader as we know it is going away in R90 snowplow/snowplow#3267.

They won't share the same configuration file going forward but I don't know if that justify having two different containers, what do you think @alexanderdean ?